platform/osx/display_server_osx.mm

```diff
@@ -848,13 +848,59 @@ - (void)mouseMoved:(NSEvent *)event {
 	ERR_FAIL_COND(!DS_OSX->windows.has(window_id));
 	DisplayServerOSX::WindowData &wd = DS_OSX->windows[window_id];
 
+	NSPoint delta = NSMakePoint([event deltaX], [event deltaY]);
+	NSPoint mpos = [event locationInWindow];
+
+	if (DS_OSX->mouse_mode == DisplayServer::MOUSE_MODE_CONFINED) {
+		// Discard late events
+		if (([event timestamp]) < DS_OSX->last_warp) {
```

Comment:
Note: for some reason clang-format is adding spaces around `->` if there's objective-c `[xxx]` expression that is not enclosed in `()` on the same line.

Reply:
Seems to be this bug report: https://bugs.llvm.org/show_bug.cgi?id=39412

Needs a squash BTW, you pushed the style fix as a new commit instead of an amend.